apps/wallet/src/components/settings/obtain-cycle/MintFromNativeToken.vue

```diff
@@ -47,13 +48,25 @@ const model = computed({
   set: value => emit('update:modelValue', value),
 });
 
+const station = useStationStore();
+
+const [icpAsset] = station.configuration.details.supported_assets.filter(asset =>
+  asset.metadata.some(
+    data => data.key === 'ledger_canister_id' && data.value === 'ryjl3-tyaaa-aaaaa-aaaba-cai',
```

Comment:
nit: maybe move those to a variable in the const file

```suggestion
    data => data.key === 'ledger_canister_id' && data.value === 'ryjl3-tyaaa-aaaaa-aaaba-cai',
```